### DISCUSSION TITLE
Any way to comment an entry in a subscription?

### DISCUSSION THREAD

**Discussion by Mosney** [AdguardTeam/AdGuardHome#2877] [Q&A] [answered]
For instance, I use a good block list but include an entry: `||ad.` , it too widely, I just want comment it rather than creat new rule: `@@||ad.$important` cause we know it will lead more advertisement escape. So anyone has good idea?

**Comment by ameshkov** [comment, marked as answer]
Add `||ad.$badfilter` to your custom filtering rules

**Reply by Mosney** [reply]
Thanks!

**Comment by Mosney** [comment]
@ameshkov Two more question, 
- Whether those `$badfilter` entries could be add in a file link and add in AdGuardHome's 'Filter'? (I think answer is TRUE, looks good for me now)
- For some `$important` entries, how to comment it? For instance,`@@||log.mmstat.com^$important`, I tried `@@||log.mmstat.com^$important$badfilter` and `@@||log.mmstat.com^$badfilter` but both failed, log.mmstat.com stll allow.

**Reply by ameshkov** [reply]
> Whether those $badfilter entries could be add in a file link and add in AdGuardHome's 'Filter'? (I think answer is TRUE, looks good for me now)

Sure, they do.

> For some $important entries, how to comment it?

```
@@||log.mmstat.com^$important
@@||log.mmstat.com^$important,badfilter
```

**Reply by Mosney** [reply]
Got it.

**Comment by Mosney** [comment]
@ameshkov Sorry to disturb again, I found hosts syntax entries looks can't comment like this method.
Eg. `127.0.0.1 minorshort.weixin.qq.com`, any idea?

**Reply by ameshkov** [reply]
Unfortunately, there's no way to do this for /etc/hosts type of blocklists.

Use AdGuard syntax when it's possible. For /etc/hosts lists - consider transforming them to AdGuard syntax automatically, see https://github.com/AdguardTeam/HostlistCompiler

**Reply by Mosney** [reply]
OK, I will try.
